Marlin/Configuration_adv.h

```diff
@@ -1262,6 +1262,7 @@
 //#define CUSTOM_USER_MENUS
 #if ENABLED(CUSTOM_USER_MENUS)
   #define USER_SCRIPT_DONE "M117 User Script Done"
+  #define USER_SCRIPT_FEEDBACK    // Disable to turn off audible feedback when user script is executed
```

Comment:
shouldn't this be USER_SCRIPT_AUDIBLE_FEEDBACK ? then you probably woudln't even need the comment